src/lib/containers/discussion_forum/ForumTable.tsx

```diff
@@ -55,6 +62,15 @@ export const ForumTable: React.FC<ForumTableProps> = ({
     filter,
   )
 
+  const onSave = (text: string, title?: string) => {
+    const request: CreateDiscussionThread = {
+      forumId: forumId,
+      title: title!,
+      messageMarkdown: text,
+    }
+    postThread(request)
+  }
```

Comment:
This logic should be moved to a ForumThreadEditor component. See other comments for details.